### ISSUE TITLE
default permission for properties ignored!

### ISSUE THREAD

**Issue by SepidehAlassi** [dhlab-basel/Knora#954]
```
### Default Object Access Permissions on ProjectMember Group
<http://rdfh.ch/permissions/0801/006-d1>

                      rdf:type knora-base:DefaultObjectAccessPermission ;

                      knora-base:forProject <http://rdfh.ch/projects/yTerZGyxjZVqFMNNKXCDPF> ;

                      knora-base:forGroup knora-base:ProjectMember ;

                      knora-base:hasPermissions "CR knora-base:Creator|M knora-base:ProjectMember|V knora-base:KnownUser|RV knora-base:UnknownUser"^^xsd:string .


### Default Object Access Permissions on beol:beolIDs
<http://rdfh.ch/permissions/0801/006-d2>

                      rdf:type knora-base:DefaultObjectAccessPermission ;

                      knora-base:forProject <http://rdfh.ch/projects/yTerZGyxjZVqFMNNKXCDPF> ;

                      knora-base:forProperty beol:beolIDs ;

                      knora-base:hasPermissions "CR knora-base:Creator|M knora-base:ProjectMember"^^xsd:string .
```

the property permission should top the default group permission.
The end result should be 
` knora-base:hasPermissions "CR knora-base:Creator|M knora-base:ProjectMember"^^xsd:string .`

**Comment by SepidehAlassi** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [make beolIDs, and id, just be visible to admin](https://trello.com/c/PCmOKNpr/112-make-beolids-and-id-just-be-visible-to-admin)

**Comment by subotic** [comment]
I've looked into this and came to the conclusion that there is no trivial way to solve this. Adding this behavior would completely change the current behavior and would require that all projects change their default object access permissions. I don't think that @loicjaouen and @mrivoal would be very pleased, as they have a few projects with very complicated permission structures.

What I would instead suggest, is that we add the possibility to provide the wanted permissions with the request for creating a resource and/or value.

Another way would be to provide the ability to change the permissions on resources and values.

Both options are anyway necessary. In any case, I'm away next week, so whatever we do, will need to wait after I'm back.

**Comment by subotic** [comment]
Yes, I saw that you added it :-) 👍  I suppose we would need this for `V1`, since I suppose she is using `V1`. Maybe the easiest would be to allow changing of the permission string for a Resource/Value.

**Comment by benjamingeer** [comment]
> Maybe the easiest would be to allow changing of the permission string for a Resource/Value.

That wouldn't be hard. I'll be back in the office (after a holiday) on 20 August and could do it then.

**Comment by subotic** [comment]
Great, thanks.
